doc/en/Docker_xpu.md

```diff
@@ -0,0 +1,92 @@
+# Intel GPU Docker Guide (Beta)
+
+## Prerequisites
+
+* Docker must be installed and running on your system.
+* Create a folder to store big models & intermediate files (e.g., /mnt/models)
+* **Before proceeding, ensure the Intel GPU driver is installed correctly on your host:** [Installation Guide](./xpu.md#1-install-intel-gpu-driver)
+
+---
+
+## Building the Docker Image Locally
+
+1. Clone the repository and navigate to the project directory:
+
+   ```bash
+   git clone https://github.com/kvcache-ai/ktransformers.git
+   cd ktransformers
+   ```
+
+2. Build the Docker image using the XPU-specific [Dockerfile.xpu](../../Dockerfile.xpu):
+
+   ```bash
+   docker build \
+      --build-arg http_proxy=.. \
+      --build-arg https_proxy=.. \
+      -t kt_xpu:0.3.1 \
+      -f Dockerfile.xpu \
+      .
+   ```
+
+---
+
+## Running the Container
+
+### 1. Start the container
+
+```bash
+docker run -td --privileged \
+    --net=host \
+    --device=/dev/dri \
+    --shm-size="16g" \
+    -v /path/to/models:/models \
+    --build-arg http_proxy=.. \
+    --build-arg https_proxy=.. \
```

Comment:
It should be `-e http_proxy=$HTTP_PROXY`

Reply:
Thanks~ Qiyuan